tests/DispatcherTest.php

```diff
@@ -34,6 +43,11 @@ function ($request, $handler) {
 
         $this->assertInstanceOf(ResponseInterface::class, $response);
         $this->assertEquals('123', (string) $response->getBody());
+
+        $response = (new Dispatcher($stack))->handle(Factory::createServerRequest('GET', '/'));
+
+        $this->assertInstanceOf(ResponseInterface::class, $response);
+        $this->assertEquals('123', (string) $response->getBody());
```

Comment:
I don't understand what do you want to test here.

Reply:
I'm missing a test testing running a dispatcher inside other dispatcher. For example:

```php
$response = Dispatcher::run([
    $middleware1,
    new Dispatcher([
        $middleware2,
        $middleware3
    ])
]);
```

Reply:
> I don't understand what do you want to test here.

I am testing that `handle` method properly delegates to `dispatch`. I guess there should be only `handle` method, but didn't want to break BC

Reply:
ah, ok